metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://angularlogos.com/assets/metadata-schema.json",
+  "logos": {
+    "angular.svg": {
+      "name": "Angular",
+      "website": "https://angular.io",
+      "license": "MIT",
```

Comment:
Looking at the presskit website: https://angular.io/presskit the license is CC BY 4.0 or something.
Or maybe just CC don't really know, should we adapt is?

Reply:
Hmm, good question. For NOW it's not really important because this is just example data but yeah, we should not forget this!

Reply:
I think we should refer to the license as said on the presskit website